src/main/java/org/frankframework/frankdoc/model/AttributesFromInterfaceRejector.java

```diff
@@ -91,7 +91,7 @@
 	 */
 	public Set<String> getRejects(FrankClass clazz) {
 		if(! implementsRejectedInterface(clazz)) {
-			log.warn("JavaDoc tag {} has no effect because [{}] does not implement [{}]", FrankElement.JAVADOC_IGNORE_TYPE_MEMBERSHIP, clazz.getName(), rejectedInterface);
+			log.error("JavaDoc tag {} has no effect because [{}] does not implement [{}]", FrankElement.JAVADOC_IGNORE_TYPE_MEMBERSHIP, clazz.getName(), rejectedInterface);
```

Comment:
Dit lijkt me geen reden om de build te laten falen

Reply:
Dit gaat over @ff.ignoreTypeMembership. Als het argument daarvan verwijst naar een interface dat niet door de geannoteerde class wordt geïmplementeerd, dan kan je de annotatie net zo goed weghalen. Het is een kleine moeite om @ff.ignoreTypeMembership weg te halen. Daarom had ik voor een error gekozen. Jullie beslissing is dus om het toch een warning te laten. Zonder tegenbericht ga ik dat doen.

Reply:
Ik heb gedaan wat jullie vragen.